EIPS/eip-5794.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 5794
+title: eth/69 - Add withdrawals to block
+description: Adds the withdrawals to the block bodies and the withdrawal root to the block header in the eth protocol
+author: Marius van der Wijden (@MariusVanDerWijden)
+discussions-to: tbd
+status: Draft
+type: Standards Track
+category: Networking
+created: tbd
+requires: 2464, 2481, 4938
+---
+
+## Abstract
+
+The [Ethereum Wire Protocol](https://github.com/ethereum/devp2p/tree/master/caps/eth.md) defines request and response messages for exchanging data between clients. The `block-bodies` definition contains the transactions and uncles of a block. We propose to extend the `block-bodies` with a list of withdrawals. The `header` definition contains the fields of a block header. We propose to extend the `header` with the `withdrawal-root`. The `block` definition contains the headers, transactions and uncles of a block. We propose to extend the `block` with a list of withdrawals.
+
+## Motivation
+With the introduction of withdrawals, the header and the block body format is changed. In order to allow nodes to respond with a valid block, the block bodies need to be extended with the withdrawal list and the block header with the withdrawal root.
+
+The `block-body` definition is used in the `BlockBodies` message. The `block` definition is used by the `NewBlock` message. The `header` definition is used in the `block` definition and the `BlockHeaders` message.
+
+## Specification
+
+Modify the following message type from the `eth` protocol:
+
+* `block-body`:
+* * **(eth/67)**: We defined `block-body = [transactions, ommers]`
```

Comment:
Could you help establish `ommers` in this doc? By either proper reference or explanation?